cgmes/src/main/java/powsybl/tutorials/cgmes/CgmesMergeTutorial.java

```diff
@@ -0,0 +1,188 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package powsybl.tutorials.cgmes;
+
+import com.powsybl.commons.io.table.AsciiTableFormatterFactory;
+import com.powsybl.commons.io.table.TableFormatterConfig;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.computation.local.LocalComputationManager;
+import com.powsybl.contingency.BranchContingency;
+import com.powsybl.contingency.ContingenciesProvider;
+import com.powsybl.contingency.Contingency;
+import com.powsybl.iidm.import_.Importers;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.util.Networks;
+import com.powsybl.loadflow.LoadFlow;
+import com.powsybl.loadflow.LoadFlowParameters;
+import com.powsybl.loadflow.LoadFlowResult;
+import com.powsybl.security.Security;
+import com.powsybl.security.SecurityAnalysis;
+import com.powsybl.security.SecurityAnalysisParameters;
+import com.powsybl.security.SecurityAnalysisResult;
+import com.rte_france.powsybl.hades2.Hades2Factory;
+import com.rte_france.powsybl.hades2.Hades2SecurityAnalysisFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.OutputStreamWriter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author Anne Tilloy <anne.tilloy at rte-france.com>
+ */
+public final class CgmesMergeTutorial {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CgmesMergeTutorial.class);
+
+    public static void main(String[] args) {
+
+        // These lines load two networks from files in CGMES format.
+        // The two networks are stored in an array list.
+        List<Network> networks = new ArrayList<Network>();
+        // A micro grid from Elia TSO (Belgium).
+        File fileBe = new File(CgmesMergeTutorial.class.getResource("/MicroGridTestConfiguration_T4_BE_BB_Complete_v2.zip").getPath());
+        Network networkBe = Importers.loadNetwork(fileBe.toString());
+        System.out.println(networkBe);
```

Comment:
Maybe more comprehensive: `System.out.println("ID of network BE: " + networkBe.getId());`